requirements.txt

```diff
@@ -1 +1,2 @@
-Django==4.2.11
\ No newline at end of file
+Django==4.2.11
+django-debug-toolbar==4.3.0
```

Comment:
Please add all dependencies. These are not enough. Check which modules are still remaining to add in the file.